### DISCUSSION TITLE
Any free API's for translation and OCR?

### DISCUSSION THREAD

**Discussion by ghost** [ogkalu2/comic-translate#136] [Ideas]
As the title, any free API's we can use for OCR and Translation?

**Comment by Brog33** [comment]
MacOS has built-in OCR, it would be nice if that could be used.

As for translation, I wish it would support local LLMs.

**Comment by ghost** [comment]
Thanks, I have discovered if your OCRing a language like French, you can just set the OCR to Default, and "English to English", and it will work.  Also, Google Translate is free, and does a "decent" translation.  Not sure if we can get free OCR for Asian, Russian, etc. languages yet.

**Reply by Brog33** [reply]
Okay, I was trying to translate from Italian to English, and Default was then GPT4-o. Nice to know.
